.github/workflows/publish.yml

```diff
@@ -45,6 +49,9 @@ jobs:
         with:
           version: 8
 
+      - name: Update npm
+        run: npm install -g npm@11.6.2
```

Comment:
Can we avoid this? I see either bumping the actions/setup-node version or using mise (which we should anyway).